.eslintrc.json

```diff
@@ -135,7 +135,9 @@
         "object-property-newline": "error",
         "one-var": ["error", "never"],
         "one-var-declaration-per-line": "error",
-        "operator-linebreak": "error",
+        "operator-linebreak": ["error", "after", {
+            "overrides": {}
```

Comment:
NB for some reason eslint has a default value that includes overrides, i.e. different behavior for ternary operators. This clears it.

Reference: http://eslint.org/docs/rules/operator-linebreak